easybuild/main.py

```diff
@@ -820,7 +826,7 @@ def main_with_hooks(args=None):
     hooks = load_hooks(eb_go.options.hooks)
 
     try:
-        main(args=args, prepared_cfg_data=(init_session_state, eb_go, cfg_settings))
+        sys.exit(int(main(args=args, prepared_cfg_data=(init_session_state, eb_go, cfg_settings))))
```

Comment:
That's a bit too heavy on the brackets (it's Python, not Lisp ;) )

```suggestion
        exit_code = main(args=args, prepared_cfg_data=(init_session_state, eb_go, cfg_settings))
        sys.exit(int(exit_code))
```